scipy/optimize/_minimize.py

```diff
@@ -38,6 +38,17 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
              callback=None, options=None):
     """
     Minimization of scalar function of one or more variables.
+     
+    In general, the optimization problems are of the form:
+    
+    minimize f(x)
+    subect to: 
```

Comment:
typo (subect), and needs double colon at the end plus a blank line on the next line. That in order to typeset the indented lines properly in compiled docs.